.github/workflows/check_atlantis.yaml

```diff
@@ -1,17 +1,3 @@
-# Copyright 2024 Hauki Tech sp. z o.o.

```

Comment:
Please adjust the message to the previously agreed format

Reply:
done